internal/pkg/apps/install_test.go

```diff
@@ -640,34 +640,33 @@ func TestInstallLocalApp(t *testing.T) {
 	mockUserID := "U001"
 
 	tests := map[string]struct {
-		isExperimental          bool
-		mockApp                 types.App
-		mockAPICreate           api.CreateAppResult
-		mockAPICreateError      error
-		mockAPIInstall          api.DeveloperAppInstallResult
-		mockAPIInstallState     types.InstallState
-		mockAPIInstallError     error
-		mockAPIUpdate           api.UpdateAppResult
-		mockAPIUpdateError      error
-		mockAuth                types.SlackAuth
-		mockAuthSession         api.AuthSession
-		mockBoltExperiment      bool
-		mockConfirmPrompt       bool
-		mockIsTTY               bool
-		mockManifest            types.SlackYaml
-		mockManifestHashInitial cache.Hash
-		mockManifestHashUpdated cache.Hash
-		mockManifestSource      config.ManifestSource
-		mockOrgGrantWorkspaceID string
-		expectedApp             types.App
-		expectedCreate          bool
-		expectedInstallState    types.InstallState
-		expectedManifest        types.AppManifest
-		expectedUpdate          bool
+		mockApp                   types.App
+		mockAPICreate             api.CreateAppResult
+		mockAPICreateError        error
+		mockAPIInstall            api.DeveloperAppInstallResult
+		mockAPIInstallState       types.InstallState
+		mockAPIInstallError       error
+		mockAPIUpdate             api.UpdateAppResult
+		mockAPIUpdateError        error
+		mockAuth                  types.SlackAuth
+		mockAuthSession           api.AuthSession
+		mockBoltExperiment        bool
+		mockBoltInstallExperiment bool
+		mockConfirmPrompt         bool
+		mockIsTTY                 bool
+		mockManifest              types.SlackYaml
+		mockManifestHashInitial   cache.Hash
+		mockManifestHashUpdated   cache.Hash
+		mockManifestSource        config.ManifestSource
+		mockOrgGrantWorkspaceID   string
+		expectedApp               types.App
+		expectedCreate            bool
+		expectedInstallState      types.InstallState
+		expectedManifest          types.AppManifest
+		expectedUpdate            bool
 	}{
-		"create a new run on slack app with a local function runtime using expected rosi defaults": {
-			isExperimental: false,
-			mockApp:        types.App{},
+		"create and install a new ROSI app with a local function runtime using expected rosi defaults when the BoltInstall experiment is disabled": {
```

Comment:
@mwbrooks Thanks for updating these test titles with the `BoltInstall` experiment status - huge help in later refactors 🪓

Reply:
@zimeg Thanks! It added some noise to the PR Review, but keeping the wording consistent will hopefully make removing the experiment a little easier.